examples/realtime/nextjs-authorization-demo/README.md

```diff
@@ -0,0 +1,161 @@
+
+> ⚠️ All the rules here are not the most secure, you should check your use case and refine them!
+#### public.profiles
+```sql
+create policy "authenticated users can view all profiles"
+on "public"."profiles"
+as PERMISSIVE for SELECT
+to authenticated
+using ( true );
+
+create policy "authenticated user can insert profile"
+on "public"."profiles"
+as PERMISSIVE for INSERT
+to supabase_auth_admin
+with check ( true );
+
+```
+#### public.rooms_users
+```sql
+create policy "authenticated users can read rooms_users"
+on "public"."rooms_users"
+as PERMISSIVE for SELECT
+to authenticated
+using ( true );
+
+create policy "authenticated can add rooms_users"
+on "public"."rooms_users"
+as PERMISSIVE for INSERT
+to authenticated
+with check ( true );
+```
+#### realtime.channels
+```sql
+create policy "authenticated users can see all channels"
+on "realtime"."channels"
+as PERMISSIVE for SELECT
+to authenticated
+using ( true );
+
+create policy "authenticated can create channel"
+on "realtime"."channels"
+as PERMISSIVE for INSERT
+to authenticated
+with check ( true );
+```
+
+#### realtime.presence
+```sql
+create policy "authenticated users can see all presences"
+on "realtime"."presences"
+as PERMISSIVE for SELECT
+to authenticated
+using ( true );
+
+create policy "authenticated can track presences"
+on "realtime"."presences"
+as PERMISSIVE for UPDATE
+to authenticated
+using ( true );
+```
+#### realtime.broadcasts
+```sql
+create policy "authenticated user in room can access"
+on "realtime"."broadcasts"
+as PERMISSIVE for SELECT
+to authenticated
+using (
+  EXISTS (
+    SELECT ru.user_id
+    FROM rooms_users ru
+    WHERE ru.user_id = auth.uid()
+    AND ru.name = realtime.channel_name()
+  )
+);
+
+create policy "authenticated user in room can broadcast"
+on "realtime"."broadcasts"
+as PERMISSIVE for UPDATE
+to authenticated
+using (
+  EXISTS (
+    SELECT ru.user_id
+    FROM rooms_users ru
+    WHERE ru.user_id = auth.uid()
+    AND ru.name = realtime.channel_name()
+  )
+);
+```
+
+## Coding concerns
+### Need to create channels
+At the moment if you are using Realtime channels you would just connect and go. If you want your channel to respect the RLS rules setup above you will need to preemptively create the channel that will be used before connecting.
+
+This can be done with a quick API call:
+```ts
+await supabase.realtime.createChannel(name)
+```
+You can check this code at the [create-room-modal.tsx](components/create-room-modal.tsx) component.
+
+### Connecting to broadcast is essentially the same
+The biggest difference from the previous way of work was really the requirement of creating the channel, other than that you can connect as usual to your Realtime channel which might fail to connect if it does not respect the RLS rules set by you.
```

Comment:
Again, should reference Broadcast and Presence specifically.

Should highlight somewhere in this README that there's two ways of creating channels for Broadcast and Presence.

One through client.channel() and the new way from client.createChannel(). It can be quite confusing.